js/pages/incidence-rates/ir-manager.js

```diff
@@ -177,30 +171,36 @@ define([
 			this.init();
 		}
 
-		async pollForInfo({silently = false} = {}) {
+		async pollForInfo({silently = false, analysisId = null} = {}) {
```

Comment:
Why do you need to pass the `analysisId` and then compare it to `selectedAnalysisId`? And then check for empty id (`if (id) {`)?

I assume that it might be related to Polling, but then you need just to `PollService.stop(this.pollTimeout);` before doing `this.pollTimeout = PollService.add(() => this.pollForInfo`, don't you?